app/backplane/radio_module/src/c_udp_listener_tenant.cpp

```diff
@@ -15,7 +15,9 @@ void CUdpListenerTenant::PostStartup() {
 
 void CUdpListenerTenant::Run() {
     NTypes::RadioBroadcastData radioBroadcastData{0};
-    // Note len argument is the size of the data buffer, not how much data to receive! rcvResult will contain the actual amount of data received or -1 on error
+
+    // Note len argument is the size of the data buffer, not how much data to receive!
```

Comment:
Is this because of how ReceiveAsynchronous works? If so this comment should be a docstring on ReceiveAsynchronous

Reply:
I don't think its necessary. I feel its a common convention in general, but I wanted to make sure its explicit to people reading the tenant code.